flow/twilio/flex/csat.yaml

```diff
@@ -0,0 +1,7 @@
+triggers:
+  - agent_command: csat
```

Comment:
add `- event_type: meya.twilio.flex.event.reservation_completed`. Not sure why this is disabling buttons. It shouldn't

Reply:
Remove the other competing `- event_type: meya.twilio.flex.event.reservation_completed` trigger from the status flow and then include the "agent has left the chat" here as well.